index.html

```diff
@@ -149,6 +156,21 @@ <h2>Attributes</h2>
                     reflected in the <a>[[\PlayoutDelayHint]]</a> internal slot.
                   </p>
                 </li>
+                <li>
+                  <p>Modifying the delay of the underlying system SHOULD affect
+                  the internal audio or video buffering gradually in order not
+                  to hurt user experience. Audio samples or video frames SHOULD
+                  be accelerated or decelerated before playout, similarly to how
+                  it is done for audio/video synchronization or in response to
+                  congestion control.</p>
```

Comment:
"similarly to how it is done" is probably a step in the right direction. This shows that there is room for improvement in specificity when it comes to acceleration and deceleration, but I don't think this PR is the place to define general principles of acceleration and deceleration. Ideally there would be an acceleration or deceleration spec already that we could just reference, since we are reusing existing mechanisms rather than invent our own. For now I like it, but I recognize the need for something like a separate acceleration/deceleration spec.

I think this PR answers the broader question about "what to do?", namely that you want a smooth transition.

Reply:
Agree, currently we have a bit fragmented picture. Acceleration or deceleration don't exist as separate concepts but similarly to jitter buffer their existence is heavily hinted from WebRTC stats. Lip sync is also a good doc as synchronization implementation originates through those SDP named groups.